fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -151,6 +302,24 @@ private fun SettingsCard(
     }
 }
 
+@Composable
+private fun highlightIcon(): Painter {
+    // As part of Bug 1841600, update this to use the correct icon based on the highlight type.
+    return painterResource(id = R.drawable.ic_shopping_cart)
+}
+
+@Composable
+private fun highlightTitleText(highlightType: HighlightType): String {
```

Comment:
The string res could be part of the HighlightType enum which would remove the need for this

Reply:
Ideally would try to avoid android/R imports in `State`. Maybe we could map them earlier similar to [ReviewGrade](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewGrade.kt#51) and how it's mapped from [Grade](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewGrade.kt#181).  The upcoming icon mapping logic would be the same, so maybe it makes more sense, wdyt?

Reply:
Sure, makes sense to be consistent with the existing method 👍